data/arweave/1TFZeEewEgUpqT5i2dsZSIRKJq3h1C7ZVi-gE8G-W6U/meta.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Evermore",
+  "symbol": "EMD_PST_TOKEN",
+  "web": "https://everemoredata.store",
+  "explorer": "https://viewblock.io/arweave/address/1TFZeEewEgUpqT5i2dsZSIRKJq3h1C7ZVi-gE8G-W6U",
+  "email": "support@evermoredata.store",
+  "links": {
+    "github": "https://github.com/MikeHibbert/evermore",
```

Comment:
Would be better if it's an account or org, didn't think people would just link to a single project but for now we can keep that

Reply:
Updated to org repo